rust-code-analysis-book/src/developers/update-grammars.md

```diff
@@ -0,0 +1,61 @@
+# Update grammars
+
+Each programming language needs to be parsed in order to extract its syntax and semantic: the so-called grammar of a language.
+In `rust-code-analysis`, we use [tree-sitter](https://github.com/tree-sitter) as parsing library since it provides a set of distinct grammars for each of our
+supported programming languages. But a grammar is not a static monolith, it changes over time, and it can also be affected by bugs,
+hence it is necessary to update it every now and then.
+
+As now, grammars can be update **only** on `Linux` and `MacOS` systems because we have used `bash` scripts to
```

Comment:
```suggestion
As now, grammars can be updated **only** on `Linux` and `MacOS` systems because we have used `bash` scripts to
```

Reply:
It's actually probably possible on Windows too, using WSL :)

Reply:
You're right, but it's not "native" eh eh
But I can add this information, thank you!